fr/orm/saving-data.rst

```diff
@@ -429,7 +433,9 @@ Par exemple, considérons le cas suivant::
             ['body' => 'Second comment', 'id' => 2],
         ]
     ];
-    $entity = $articles->newEntity($data);
+    
+    $article = $articles->newEntity($data);
+    $articles->save($article);
```

Comment:
If we don't save, 'Second comment' with id 2 isn't created. And the next sentence make no sense.